browser-test/src/api_docs.test.ts

```diff
@@ -124,10 +118,8 @@ describe('Viewing API docs', () => {
     // Select minimal sample program so the screenshot will be smaller.
     await page.selectOption('#select-slug', {value: 'minimal-sample-program'})
 
+    // Opening the accordion
```

Comment:
question (non-blocking): why do we no longer need the `waitForTimeout` calls?

Reply:
Ohh maybe because with the USWDS classes it's all done with css now?

Reply:
The `waitForTimeout` calls were because of the transition time it took for the "^" icon to spin when we open/closed the accordion.  With USWDS, they just swap "+" for "-" or vice versa, without transition time.